### PR TITLE
FIX-1861 Temporarily hide fund mapping till the feature is complete

### PR DESCRIPTION
**Pull request by Onyx2406**
## Description
Temporarily hides fund mapping till the feature is complete A conditional Angular directive (*ngIf="!shouldShowFundMapping") is used to hide the "Fund Mapping" section based on the shouldShowFundMapping variable.

## Related issues and discussion
#1861

## Screenshots, if any
image

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by Onyx2406** [comment]
@adamsaghy

File: src/app/organization/organization.component.html [hunk 1]
```diff
@@ -100,6 +100,16 @@ <h4 matLine>Password Preferences</h4>
               <p matLine>Define standards for enforcing the usage of stronger passwords</p>
             </mat-list-item>
           </div>
+
```
Comment (adamsaghy) [inline src/app/organization/organization.component.html:103]: @Onyx2406 Is there a reason why the bulkimport was moved?

File: src/app/organization/organization.component.html [hunk 2]
```diff
@@ -49,7 +49,7 @@ <h4 matLine>Standing Instructions History</h4>
             </mat-list-item>
           </div>
 
-          <div #fundmapping>
+          <div #fundmapping *ngIf="!shouldShowFundMapping">
```
Comment (adamsaghy) [inline src/app/organization/organization.component.html:52]: @Onyx2406 This variable name and condition seems wrong to me:
- if the `shouldShowFundMapping` variable is `true` it should show the fund mapping, but here it doing exactly the opposite.

What do you think?

**Review by adamsaghy — CHANGES_REQUESTED** [review]
Please kindly review my questions!

File: src/app/organization/organization.component.html (hunk 1 above)
Comment (Onyx2406) [reply src/app/organization/organization.component.html:103]: @adamsaghy I've moved "bulkimport" from the right to the left to ensure frontend consistency. Let me know if you'd like me to move it back to its original position, but if I do, the left block will remain blank.
Check this: https://i.imgur.com/HOETIdL.png

File: src/app/organization/organization.component.html (hunk 2 above)
Comment (Onyx2406) [reply src/app/organization/organization.component.html]: @adamsaghy right, I have fixed it now.

File: src/environments/.env.ts [hunk 3]
```diff
@@ -2,8 +2,8 @@
 /* tslint:disable */
 export default {
   'mifos_x': {
-    'version': '0.0.0',
```
Comment (adamsaghy) [inline src/environments/.env.ts:5]: do we need these changes?

**Review by adamsaghy — APPROVED** [review]
LGTM

File: angular.json [hunk 4]
```diff
@@ -229,5 +229,8 @@
         }
       }
     }
+  },
```
[the remark this answers is not in the document]
Comment (Onyx2406) [reply angular.json]: @adamsaghy I didn't make these changes, looks like auto generated.

File: src/environments/.env.ts (hunk 3 above)
Comment (Onyx2406) [reply src/environments/.env.ts]: @adamsaghy I didn't make these changes, looks like auto generated.

**Pull request merged**